packages/babel-runtime/helpers/esm/iterableToArray.js

```diff
@@ -1,3 +1,11 @@
 export default function _iterableToArray(iter) {
-  if (Symbol.iterator in Object(iter) || Object.prototype.toString.call(iter) === "[object Arguments]") return Array.from(iter);
```

Comment:
Did you manually edit this file? You should first update the helper in `babel-helpers`, and then generate this file (I think that it is done on `make bootstrap`)

Reply:
ah got it, will update this PR

Reply:
ok, just updated this, changed `packages/babel-helpers/src/helpers.js`  with the correct syntax and running `make bootstrap` updated all the relevant runtime helpers. Thank you~